app/notes/page.tsx

```diff
@@ -1,58 +1,48 @@
-import Link from "next/link";
+'use client';
 import CreateNote from "@/app/notes/CreateNote";
-import { RefreshOnFocus } from "./refreshOnFocus";
-import pb from "@/app/lib/pocketbase"
+import {RefreshOnFocus} from "./refreshOnFocus";
+import {getNotes, Note} from "./getNotes"
+import {useEffect, useState} from "react";
+import {useRouter} from "next/navigation";
 
 // this is needed beacause I don't use fetch, so I have to specify the caching method
+
 export const dynamic = 'auto',
-    dynamicParams = true,
-    revalidate = 0,
-    fetchCache = 'auto',
-    runtime = 'nodejs',
-    preferredRegion = 'auto'
+    dynamicParams = true
 
-async function getNotes() {
-    // const res = await fetch('http://127.0.0.1:8090/api/collections/notes/records?page&perPage',
-    //     {cache: "no-cache"});
-    // const data = await res.json();
-    try {
-        const data = await pb.collection('notes').getList();
-        return data?.items as any[];
-    }catch(error){
-        console.error("habar n am");
-    }
-}
+export default function NotesPage() {
+    const [notes, setNotes] = useState<any[]>([]);
+    const [user, setUser] = useState<string | null>(null);
+    const router = useRouter();
+
+    useEffect(() => {
```

Comment:
could be done in a spearate hook I think, also, why are you not storing the user in a persisted global storage?

Reply:
cuz idk how to do that. i ll look into it